### PR TITLE
EHPR-74 | Restructure Project

### PR DESCRIPTION
**Pull request by fwpushan**
Restructured project dir.

### CODE REVIEW

**Review by hitankar-freshworks — APPROVED** [review]

File: .yarnrc.yml [hunk 1]
```diff
@@ -1,2 +1,9 @@
-yarnPath: .yarn/releases/yarn-3.1.0.cjs
+nmHoistingLimits: none
+
 nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@ojkelly/plugin-build.cjs
```
Comment (arranfw) [inline .yarnrc.yml:6]: What is this piece for?

File: .gitignore [hunk 2]
```diff
@@ -15,3 +15,6 @@ node_modules
 
 .eslintcache
 .pgdata
+.build
+yarn.build.json
+packing.sh
```
Comment (arranfw) [inline .gitignore:20]: what is this shell script we're ignoring?

**Review by arranfw — APPROVED** [review]
I think there are some references in Docker that will need updating

File: .gitignore (hunk 2 above)
Comment (fwpushan) [reply .gitignore:20]: it is WIP

File: .yarnrc.yml (hunk 1 above)
Comment (fwpushan) [reply .yarnrc.yml:6]: I added one yarn plugin to build and bundle ..... but bundle not working as expected but build is good.

**Pull request merged**